### DISCUSSION TITLE
thread '<unnamed>' panicked at 'Once has panicked', <::core::macros::panic macros>:2:2

### DISCUSSION THREAD

**Discussion by 815943555** [filecoin-project/lotus#3433] [Storage Provider]
2020-09-01T03:59:18.338 INFO bellperson::gpu::locks > GPU is available for Multiexp!
3|lotus miner  | thread '<unnamed>' panicked at 'Once has panicked', <::core::macros::panic macros>:2:2
3|lotus miner  | stack backtrace:
3|lotus miner  |    0:          0x20a4b94 - backtrace::backtrace::libunwind::trace::h5d52ba5f20882f09
3|lotus miner  |                                at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.44/src/backtrace/libunwind.rs:86
3|lotus miner  |    1:          0x20a4b94 - backtrace::backtrace::trace_unsynchronized::hceee092869668a74
3|lotus miner  |                                at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.44/src/backtrace/mod.rs:66
3|lotus miner  |    2:          0x20a4b94 - std::sys_common::backtrace::_print_fmt::ha312c2904605e4d5
3|lotus miner  |                                at src/libstd/sys_common/backtrace.rs:78
3|lotus miner  |    3:          0x20a4b94 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h5b9981092140b727
3|lotus miner  |                                at src/libstd/sys_common/backtrace.rs:59
3|lotus miner  |    4:          0x20d8b9c - core::fmt::write::h5f6d7d8de88b4173
3|lotus miner  |                                at src/libcore/fmt/mod.rs:1063
3|lotus miner  |    5:
3|lotus miner  |         0x20968b3 - std::io::Write::write_fmt::h893169117de3cc15
3|lotus miner  |                                at src/libstd/io/mod.rs:1426
3|lotus miner  |    6:          0x20a99c5 - std::sys_common::backtrace::_print::h8ab61d4120f7a335
3|lotus miner  |                                at src/libstd/sys_common/backtrace.rs:62
3|lotus miner  |    7:          0x20a99c5 - std::sys_common::backtrace::print::h8aae19fbb153bf2a
3|lotus miner  |                                at src/libstd/sys_common/backtrace.rs:49
3|lotus miner  |    8:          0x20a99c5 - std::panicking::default_hook::{{closure}}::h1ee5b7d8b6f83429
3|lotus miner  | 
3|lotus miner  |                          at src/libstd/panicking.rs:204
3|lotus miner  |    9:          0x20a9712 - std::panicking::default_hook::hd6c32c13403f9210
3|lotus miner  |                                at src/libstd/panicking.rs:224
3|lotus miner  |   10:          0x20aa022 - std::panicking::rust_panic_with_hook::h1f2449d529a25f22
3|lotus miner  |                                at src/libstd/panicking.rs:470
3|lotus miner  |   11:          0x20a9c0b - rust_begin_unwind
3|lotus miner  |                                at src/libstd/panicking.rs:378
3|lotus miner  |   12:          0x20d5831 - core::panicking::panic_fmt::h2c05f81b567d1861
3|lotus miner  | 
3|lotus miner  |                       at src/libcore/panicking.rs:85
3|lotus miner  |   13:          0x20d577d - core::panicking::panic::h8a9eda1e10298363
3|lotus miner  |                                at src/libcore/panicking.rs:52
3|lotus miner  |   14:          0x1f2be1c - spin::once::Once<T>::call_once::h0e78d228133a835f
3|lotus miner  |   15:          0x1f250b4 - bellperson::gpu::utils::get_core_count::h19590e62d83bf43a
3|lotus miner  |   16:          0x1b8bd56 - bellperson::gpu::multiexp::SingleMultiexpKernel<E>::create::h2133b3ed7578a78e
3|lotus miner  |   17:          0x1aa2e4e - <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::try_fold::h8e5c27dd6c08077c
3|lotus miner  |   18:          0x22a33c7 - <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter::hdea0806bec1f1f00
3|lotus miner  |   19:          0x1b8ae5a - bellperson::gpu::multiexp::MultiexpKernel<E>::create::h01f5ece5270cb4be
3|lotus miner  |   20:          0x22bb2ea - bellperson::multiexp::create_multiexp_kernel::h8ef01f3800b6edda
3|lotus miner  |   21:          0x22ce083 - bellperson::gpu::locks::LockedMultiexpKernel<E>::init::hba526ca0b72824ca
3|lotus miner  |   22:          0x22cf016 - bellperson::gpu::locks::LockedMultiexpKernel<E>::with::hd4d61895d790cbe2
3|lotus miner  |   23:          0x22bb4f4 - bellperson::multiexp::multiexp::h195ffe8dbe89f7d8
3|lotus miner  |   24:
3|lotus miner  |    0x1a5494e - <core::iter::adapters::ResultShunt<I,E> as core::iter::traits::iterator::Iterator>::next::h4d15a8500a802007
3|lotus miner  |   25:          0x229a8ec - <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter::h8271b89d0e040900
3|lotus miner  |   26:          0x1a8c414 - core::iter::adapters::process_results::haa1a88cfed7a6140
3|lotus miner  |   27:          0x1be5f2d - bellperson::groth16::prover::create_proof_batch_priority_inner::hcbe8479d44248d29
3|lotus miner  | 
3|lotus miner  | 28:          0x1b80b53 - std::panicking::try::do_call::hffbba877f57cd7f1
3|lotus miner  |   29:          0x20bb387 - __rust_maybe_catch_panic
3|lotus miner  |                                at src/libpanic_unwind/lib.rs:86
3|lotus miner  |   30:          0x25163d1 - <rayon_core::job::StackJob<L,F,R> as rayon_core::job::Job>::execute::hdc81cc8ab11fd1c4
3|lotus miner  |   31:          0x2021d2e - rayon_core::registry::WorkerThread::wait_until_cold::h4dc4de3e2d1af613
3|lotus miner  |   32:          0x2020424 - rayon_core::registry
3|lotus miner  | ::ThreadBuilder::run::h5313262957692f77
3|lotus miner  |   33:          0x2028dcf - std::sys_common::backtrace::__rust_begin_short_backtrace::h5d84e7f6f38b5ee2
3|lotus miner  |   34:          0x20229be - std::panicking::try::do_call::hb9ae843482d0510d
3|lotus miner  |   35:          0x20bb387 - __rust_maybe_catch_panic
3|lotus miner  |                                at src/libpanic_unwind/lib.rs:86
3|lotus miner  |   36:          0x2029113 - core::ops::function::FnOnce::call_once{{vtable.shim}}::h94e4e276f65f5cf3
3|lotus miner  |   37:          0x208635f - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::h36e46972b1b7f7b7
3|lotus miner  |                                at /rustc/8d69840ab92ea7f4d323420088dd8c9775f180cd/src/liballoc/boxed.rs:1017
3|lotus miner  |   38:          0x20b9ccd - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::hdde89c32ce6999de
3|lotus miner  |                                at /rustc/8d69840ab92ea7f4d323420088dd8c9775f180cd/src/liballoc/boxed.rs:1017
3|lotus miner  |   39
3|lotus miner  | :          0x20b9ccd - std::sys_common::thread::start_thread::h5ae3fdd4dad03dd5
3|lotus miner  |                                at src/libstd/sys_common/thread.rs:13
3|lotus miner  |   40:          0x20b9ccd - std::sys::unix::thread::Thread::new::thread_start::h76b2403b2434c4c0
3|lotus miner  |                                at src/libstd/sys/unix/thread.rs:80
3|lotus miner  |   41:     0x7f054b9d16db - start_thread
3|lotus miner  |   42:     0x7f054b4e2a3f - __clone
3|lotus miner  |   43:                0x0 - <unknown>
3|lotus miner  | 2020-08-31T19:59:19.470Z	ERROR	storageminer	storage/wdpost_run.go:60	runPost failed: running post failed:
3|lotus miner  |     github.com/filecoin-project/lotus/storage.(*WindowPoStScheduler).runPost
3|lotus miner  |         /home/jack/go_workspace/lotus/storage/wdpost_run.go:409
3|lotus miner  |   - Rust panic: no unwind information
3|lotus miner  |     github.com/filecoin-project/filecoin-ffi.GenerateWindowPoSt
3|lotus miner  |     	/home/jack/go_workspace/lotus/extern/filecoin-ffi/proofs.go:579
3|lotus miner  |     github.com/filecoin-project/lotus/extern/sector-storage/ffiwrapper.(*Sealer).GenerateWindowPoSt
3|lotus miner  |     	/home/jack/go_workspace/lotus/extern/sector-storage/ffiwrapper/verifier_cgo.go:41
3|lotus miner  |     github.com/filecoin-project/lotus/storage.(*WindowPoStScheduler).runPost
3|lotus miner  |     	/home/jack/go_workspace/lotus/storage/wdpost_run.go:407
3|lotus miner  |     github.com/filecoin-project/lotus/storage.(*WindowPoStScheduler).doPost.func1
3|lotus miner  |     	/home/jack/go_workspace/lotus/storage/wdpost_run.go:49
3|lotus miner  |     runtime.goexit
3|lotus miner  |     	/usr/local/go/src/runtime/asm_amd64.s:1373
3|lotus miner  | 2020-08-31T19:59:19.470Z	ERROR	storageminer	storage/wdpost_run.go:29	TODO

**Comment by jennijuju** [comment]
Seems like need help with trouble shooting, however, provided information is not enough to define an issue. Transfer to discussion.
